docs/examples/reftests.jl

```diff
@@ -49,7 +49,7 @@ g = path_graph(4)
 elabels = ["𐄂" for i in 1:ne(g)]
 elabels_align = (:center, :center)
 fig, ax, p = graphplot(g; layout=SquareGrid(), elabels, elabels_align,
-                       elabels_color=:red)
+                       elabels_color=:red, elabels_rotation=nothing)
```

Comment:
```suggestion
                       elabels_color=:red)
```
i think i'd prefer to update the ref images, `elabels_rotation=nothing` works but seems kinda useless :D

Reply:
I agree that the ref images should be updated.
And since `elabels_rotation=automatic` is becoming the default, probably that's how most ref images should be compiled.
Adding a ref image with `elabels_rotation=nothing` would still make sense though.